### PR TITLE
Change person to event

### PR DESCRIPTION
**Pull request by glozxi**
Contributes to #106

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -138,12 +138,12 @@ How the parsing works:
...
 * stores a `UserPref` object that represents the user’s preferences. This is exposed to the outside as a `ReadOnlyUserPref` objects.
 * does not depend on any of the other three components (as the `Model` represents data entities of the domain, they should make sense on their own without depending on other components)
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `Person` needing their own `Tag` objects.<br>
+<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Event` references. This allows `AddressBook` to only require one `Tag` object per unique tag, instead of each `Event` needing their own `Tag` objects.<br>
```
Comment (bryansendeavour) [inline docs/DeveloperGuide.md:146]: Do we want to keep it as AddressBook or change it to EventBook?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -320,15 +320,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`   | User                 | View timing of upcoming events                   | I know when to go                                    |
 | `* *`   | User                 | See the most urgent events                       | I know what to do first                              |
 | `* *`   | Careless user        | Undo accidental deletion                         | I can undo deleted events                            |
-| `* *`   | Impatient person     | Search contacts                                  | I can quick find specific POC                        |
-| `* *`   | Impatient person     | Search events                                    | I can quick find specific event                      |
-| `* *`   | Lazy person          | Filter events to find similar events             | I can find similar events easily                     |
+| `* *`   | Impatient event     | Search contacts                                  | I can quick find specific POC                        |
```
Comment (bryansendeavour) [inline docs/DeveloperGuide.md:323]: Maybe this is an accident made when renaming?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -320,15 +320,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+| `* *`   | Lazy event          | Filter events to find similar events             | I can find similar events easily                     |
 | `* *`   | User                 | Add all information to an event at the same time | I can save time on adding all the information        |
 | `* *`   | User                 | View events associated to each contact           | I know what events to contact the POC for            |
 | `*`      | User                 | Toggle between light and dark mode               | I can use the theme i prefer                         |
 | `*`      | User                 | Redo previous action                             | I can redo previous action                           |
 | `*`      | User                 | Use abbreviation                                 | I can quickly insert items                           |
-| `*`      | Goal oriented person | Set goals                                        | I can look at my goals                               |
+| `*`      | Goal oriented event | Set goals                                        | I can look at my goals                               |
```
Comment (EksdeeeX) [inline docs/DeveloperGuide.md:331]: Perhaps this could be left as person instead?

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -685,17 +685,17 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Deleting a event
```
Comment (bryansendeavour) [inline docs/DeveloperGuide.md:688]: Maybe this should be `an` instead of `a`

File: docs/UserGuide.md [hunk 5]
```diff
@@ -182,8 +182,8 @@ Edits the specified event from the event book.
   - Tags can be removed by typing `t/` without specifying any tags after it.
 
 **Example**:
-  - `edit 1 p/91234567 r/100` Edits the phone number and rate of the 1st person to be 91234567 and j100 respectively.
-  - `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be Betsy Crower and clears all existing tags.
+  - `edit 1 p/91234567 r/100` Edits the phone number and rate of the 1st event to be 91234567 and j100 respectively.
```
Comment (bryansendeavour) [inline docs/UserGuide.md:185]: I don't think our edit function works like this. Phone number/ Contact cannot be edited this way.

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (glozxi) [reply docs/DeveloperGuide.md:331]: 👍

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (glozxi) [reply docs/DeveloperGuide.md:323]: Yeah, I'll fix it

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (glozxi) [reply docs/DeveloperGuide.md:688]: 👍 I'll correct it

File: docs/UserGuide.md (hunk 5 above)
Comment (glozxi) [reply docs/UserGuide.md:185]: I'll replace this line with:

`edit 1 r/100` Edits the rate of the 1st event to be 100.

Comment (bryansendeavour) [reply docs/UserGuide.md:185]: 👍  Good that we catch this mistake from previous implementations through this renaming PR

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (glozxi) [reply docs/DeveloperGuide.md:146]: 👍 Will change to EventBook

**Review by bryansendeavour — APPROVED** [review]
LGTM

**Pull request merged**